extensions/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -234,7 +234,7 @@ const Layout: React.FC<unknown> = () => {
             </Stack>
           </Stack>
 
-          <Stack align="center" justify="between" customStyle="sticky top-0 h-screen">
+          <Stack align="center" justify="between" customStyle="sticky top-0 h-screen relative">
```

Comment:
relative and sticky are mixed here, I understand the relative part is a left over from your local test.